src/openms/source/CHEMISTRY/TheoreticalSpectrumGenerator.cpp

```diff
@@ -674,28 +685,31 @@ namespace OpenMS
           const AASequence ion = peptide.getSuffix(i);
           addIsotopeCluster_(spectrum, ion, ion_names, charges, res_type, charge, intensity);
         }
-      }
+        chunks.add(true);
 
-      if (add_losses_ && add_isotopes_) // otherwise losses are already added, see above
```

Comment:
Why can you remove add_isotopes_ here as a condition?

Reply:
As the if-statement is nested into the `if(add_isotopes_ )`, we don't have to check it anymore, as it must already be true.